content/pages/test_demo_landing.en-US.page

```diff
@@ -66,7 +66,7 @@
       "type": "call-to-action",
       "id": "calltoactionsNDR",
       "title": "Book a free ecommerce consulting session",
-      "background": null,
+      "background": "bg-gray-800",
```

Comment:
bad idea to use tailwind classes in json, it should be HEX code, like "\#DDDDDD"

Reply:
We could select only one of the predefined  in _sections.json

The context is the Page Builder
As a user we can see variants in the settings tab in the preview mode
![image](https://github.com/VirtoCommerce/vc-theme-b2b-vue/assets/19205780/f913dccc-79cc-4bb3-9db2-66515bd5746d)

Reply:
ok, use new color palette variables then.